src/main/kotlin/cli/Parse.kt

```diff
@@ -64,4 +65,22 @@ internal class Parse : CliktCommand(name = "parse") {
             outputFile.writeToFileOrStdout(JsonMapper.toJson(output, prettyPrint))
         }
     }
+
+    private class School : CliktCommand(name = "school") {
```

Comment:
I think we might want to break this into two separate commands. One for schools, and another for subjects. And then maybe one for both? Because I realize that they both use the same input data.